### PR TITLE
Changes_GOP-877,GOP-876,GOP-847-BUG-minimum_height_for_textarea

### PR DESCRIPTION
**Pull request by adhiyan-tangoe**
Fixed GOP-876 uniform card margins, GOP-877 set minimum_height_for_textarea and GOP-847 file uploading component bug fix.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Style Update (CSS)
- [ ] Other... Please describe:

## What is the current behavior?

Resolves 

#877 
#876 
#847 

## What is the new behavior?
#877 - Textarea will have the minimum width as 70px.
#876 - Last child of the card content will have no margin bottom.
#847 - Reselecting the same file after clearing is working fine.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [inline projects/go-lib/styles/_forms.scss:2]: In stylesheet, we generally go with single quoted strings.

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [inline projects/go-lib/styles/_forms.scss:4]: Across this repo, the decimal values has been beginning with ```.```. Let's keep consistent with practice.

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [inline projects/go-lib/styles/_forms.scss:57]: Same as above.

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [inline projects/go-lib/styles/_forms.scss:143]: Same as above

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [inline projects/go-lib/styles/_forms.scss:148]: Same as above

**Review by jiaweicai92 — CHANGES_REQUESTED** [review]
The fix looks good. Left some comments on changes on ```_forms.scss```. Great job 👍

**Review by swimtech20 — DISMISSED** [review]
Nice work!

File: projects/go-lib/styles/_forms.scss
Comment (swimtech20) [reply projects/go-lib/styles/_forms.scss]: @jiaweicai92 would it be a good idea to switch practice on this? I know it's minor but I think having a 0 in front makes it a bit easier to read (and aesthetically pleasing, in my opinion). Like I said, this is a minor decision so I don't _really_ care but just a thought.

File: projects/go-lib/styles/_forms.scss
Comment (swimtech20) [reply projects/go-lib/styles/_forms.scss]: I agree.

**Review by jiaweicai92 — DISMISSED** [review]
👍 Looks great!

File: projects/go-lib/styles/_forms.scss
Comment (jiaweicai92) [reply projects/go-lib/styles/_forms.scss]: @swimtech20, I personally prefer your and @adhiyan-tangoe's approach. Yet, let's keep the current format, for the sake of consistency. We could create a tech debt / followup task for refactoring the repo.

Comment (adhiyan-tangoe) [reply projects/go-lib/styles/_forms.scss]: @jiaweicai92 That sounds good to me.

**Comment by stolve** [comment]
@kpmurphh and myself have no way to provide approval/feedback for this issue. We do not have an instance of goponents on our systems in order to look at the visual fix and we are not in a position to give feedback on coding approaches. Please attach a screenshot of the update or block time on one of our calendars for a review session.

File: projects/go-lib/styles/_forms.scss
Comment (swimtech20) [inline projects/go-lib/styles/_forms.scss:101]: This style is for `min-height` but the conversation/testing instructions say `min-width` of 70px. Just want to make sure this is the correct change...

Comment (swimtech20) [reply projects/go-lib/styles/_forms.scss:101]: @adhiyan-tangoe I am not questioning the requirements of this ticket. The description above says "width":

Screen Shot 2023-03-31 at 7 41 34 AM

Just making sure the description is wrong, not the actual style...

Comment (stolve) [reply projects/go-lib/styles/_forms.scss:101]: Its supposed to be height not width just to clarify from UX. The image in the initial ticket shows the height issue.

**Comment by stolve** [comment]
looks good to me

**Review by stolve — DISMISSED** [review]
Based on the netlify link the change looks good to me.

File: projects/go-lib/styles/_forms.scss
Comment (swimtech20) [reply projects/go-lib/styles/_forms.scss:101]: @stolve thank you! Approving now.

**Review by swimtech20 — APPROVED** [review]

**Review by swimtech20 — APPROVED** [review]

**Pull request merged**
